mslib/msui/_tests/test_topview.py

```diff
@@ -203,6 +203,32 @@ def test_move_point(self, mockbox):
         assert len(self.window.waypoints_model.waypoints) == 4
         assert mockbox.critical.call_count == 0
 
+    @mock.patch("mslib.msui.mss_qt.QtWidgets.QMessageBox")
+    def test_connect_point(self, mockbox):
+        """
+        Test connecting the last and first point
+        Test connecting the first point to itself
+        """
+        count = len(self.window.waypoints_model.waypoints)
+
+        #Test if the last waypoint connects to the first
```

Comment:
I'm currently not happy with the flake8 checker, usually there should be a a blank after a comment sign

https://www.python.org/dev/peps/pep-0008/#comments